src/plugins/discover/public/application/view_components/panel/index.tsx

```diff
@@ -4,18 +4,17 @@
  */
 
 import React from 'react';
-import ReactDOM from 'react-dom';
-import { ViewMountParameters } from '../../../../../data_explorer/public';
+import { ViewProps } from '../../../../../data_explorer/public';
 import { OpenSearchDashboardsContextProvider } from '../../../../../opensearch_dashboards_react/public';
-import { DiscoverServices } from '../../../build_services';
+import { Panel } from './panel';
+import { getServices } from '../../../opensearch_dashboards_services';
 
-export const renderPanel = ({ panelElement }: ViewMountParameters, services: DiscoverServices) => {
-  ReactDOM.render(
+// eslint-disable-next-line import/no-default-export
+export default function PanelApp(props: ViewProps) {
```

Comment:
This may be temp - otherwise should include an explanation of why it's necessary.

Reply:
I will add this in a followup, but the reason is that to lazy load components, the syntax requires default components to be exported.